nailgun/entities.py

```diff
@@ -4153,6 +4165,62 @@ def upload_content(self, synchronous=True, **kwargs):
             )
         return json
 
+    def remove_content(self, synchronous=True, **kwargs):
+        """Remove content from a repository
+
+        It expects packages/puppet modules/docker manifests ids sent as data.
+        Here is an example of how to use this method::
+
+            repository.remove_content(data={'ids': [package.id]})
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all JSON decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+
+        """
+        kwargs = kwargs.copy()
+        kwargs.update(self._server_config.get_client_kwargs())
+        response = client.put(self.path('remove_content'), **kwargs)
+        return _handle_response(response, self._server_config, synchronous)
+
+    def list_puppet_modules(self, synchronous=True, **kwargs):
```

Comment:
As far as I remember the helper names are after the API path. Shouldn't this be named `puppet_modules` instead since the path is `puppet_modules` and not `list_puppet_modules`?